module/VuFind/src/VuFind/Search/EDS/ResultsFactory.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * Factory for EDS search results objects.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see
+ * <https://www.gnu.org/licenses/>.
+ *
+ * @category VuFind
+ * @package  EBSCO
+ * @author   Maccabee Levine <msl321@lehigh.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+
+namespace VuFind\Search\EDS;
+
+use Laminas\ServiceManager\Exception\ServiceNotCreatedException;
+use Laminas\ServiceManager\Exception\ServiceNotFoundException;
+use Psr\Container\ContainerExceptionInterface as ContainerException;
+use Psr\Container\ContainerInterface;
+use VuFind\Config\ConfigManagerInterface;
+
+/**
+ * Factory for EDS search results objects.
+ *
+ * @category VuFind
+ * @package  EBSCO
+ * @author   Maccabee Levine <msl321@lehigh.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class ResultsFactory extends \VuFind\Search\Results\ResultsFactory
+{
+    /**
+     * Config filename
+     *
+     * @var string
+     */
+    protected string $configName = 'EDS';
```

Comment:
This is just future-proofing, from lessons learned on Blender2, in case we want to extend it in the future for EPF or whatever else.  Currently EPF has no checks on the search query, limiters, etc.  @cwolfebsco if that should change, please let me know and we can address it separately.

Reply:
@maccabeelevine - In `/edsapi/publication/search` submitting a limiter only query would yield an error 103. Best of my knowledge query is a required parameter for publication search

```
  "DetailedErrorDescription": "Required parameter: query is missing",
  "ErrorDescription": "Missing Parameter",
  "ErrorNumber": "103"
```

I might have misunderstood the question though :) Happy New Year!